packages/networks/bin/utils/contracts.ts

```diff
@@ -2,8 +2,12 @@ import { ethers } from 'ethers'
 import { queryGraph } from './subgraph'
 import { HookType } from '@unlock-protocol/types'
 
+// importing from contracts package will create recusrive deps conflict
```

Comment:
HA! I guess the good thing is that this script will fail when we uograde contracts!